iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/DigitalTwinClientTests.java

```diff
@@ -253,29 +253,6 @@ public void getDigitalTwinWithAzureSasCredential() {
         assertEquals(responseWithHeaders.body().getMetadata().getModelId(), E2ETestConstants.THERMOSTAT_MODEL_ID);
     }
 
-    @Ignore("RBAC authentication isn't supported on hub yet")
```

Comment:
As mentioned in the description of this PR, the service won't support PnP APIs yet, so the SDK has been stripped of it's TokenCredential digital twin APIs